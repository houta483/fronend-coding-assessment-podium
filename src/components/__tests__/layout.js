import React from "react"
import renderer from "react-test-renderer"

import Layout from "../layout"

describe("Card", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Layout siteTitle="Default Starter" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})