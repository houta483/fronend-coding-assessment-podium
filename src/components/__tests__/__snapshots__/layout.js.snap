// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<div>
  <div
    className="flex-center space-around border header-lg"
  >
    Shakespeare's Reviews
  </div>
</div>
`;
